ecs_fargate/README.md

```diff
@@ -91,6 +101,8 @@ aws ecs run-task --cluster <CLUSTER_NAME> \
 ### Metric Collection
 After the Datadog Agent is setup as described above, the [ecs_fargate check][10] collects metrics with autodiscovery enabled. Add Docker labels to your other containers in the same task to collect additional metrics.
 
+Metrics collected via the agent will be namespaced as `ecs.fargate.*`.
```

Comment:
Isn't this covered by the . metric metadata below?

Reply:
Yes it is.  Removed the line.